modules/dynamic-plugins/proc-install-third-party-plugins-rhdh.adoc

```diff
@@ -0,0 +1,120 @@
+[id="proc-install-third-party-plugins-rhdh_{context}"]
+= Installing third-party plugins in {product}
+
+You can install a third-party plugins in {product} without rebuilding the {product-very-short} application.
+
```

Comment:
The following information is important, and it should not be omitted. 

```suggestion
The placement of dynamic-plugin-config.yaml depends on the deployment method. For more information, see xref:proc-config-dynamic-plugins-rhdh-operator_{context}[] and xref:con-install-dynamic-plugin-helm_{context}[]


Plugins are defined in the `plugins` array in the `dynamic-plugin-config.yaml` file. Each plugin is defined as an object with the following properties:

* `package`: The package definition of the plugin. This can be an OCI image, `tgz` archive, npm package, or a directory path.
* `disabled`: A boolean value that determines whether the plugin is enabled or disabled.
* `integrity`: The integrity hash of the package. This is required for `tgz` archives and npm packages.
* `pluginConfig`: The configuration for the plugin. For backend plugins this is optional. For frontend plugins this is required. This is fragment o the `app-config.yaml` file. Anything that is added to this object will be merged with the main app-config.yaml file.
```